webui/portal/src/main/java/org/exoplatform/portal/webui/workspace/UIPortalApplication.java

```diff
@@ -812,21 +812,21 @@ public void processDecode(WebuiRequestContext context) throws Exception {
     @Override
     public void processAction(WebuiRequestContext context) throws Exception {
         PortalRequestContext pcontext = (PortalRequestContext) context;
-        // String requestURI = pcontext.getRequestURI();
         RequestNavigationData requestNavData = pcontext.getNavigationData();
 
         boolean isAjax = pcontext.useAjax();
 
         if (!isAjax) {
-            if (isAjaxInLastRequest) {
-                isAjaxInLastRequest = false;
-                if (requestNavData.equals(lastNonAjaxRequestNavData) && !requestNavData.equals(lastRequestNavData)) {
-                    NodeURL nodeURL = pcontext.createURL(NodeURL.TYPE).setNode(getCurrentSite().getSelectedUserNode());
-                    pcontext.sendRedirect(nodeURL.toString());
-                    return;
-                }
+          if (isAjaxInLastRequest) {
+            isAjaxInLastRequest = false;
+            if (requestNavData.equals(lastNonAjaxRequestNavData) && !requestNavData.equals(lastRequestNavData)
+                && pcontext.getPortletParameters().isEmpty()) {
```

Comment:
Could you tell more about the relation between the request parameters and the redirection ? why did you add the check on the portlet parameters ?